src/main/java/entities/Person.java

```diff
@@ -26,6 +31,13 @@
     private Integer id;
     private String email, firstName, lastName;
 
+    @ManyToMany(fetch = FetchType.LAZY, cascade = CascadeType.PERSIST)
+    @JoinColumn(name = "hobby_id")
+    private List<Hobby> hobbies;
+
+    public Person()
+    {}
```

Comment:
This is gonna die a alt+shift+f death some day 😄

Reply:
Yeah lol, we'll never agree on a standard way of formatting things :P